lib/evss/documents_service.rb

```diff
@@ -13,5 +13,20 @@ def initialize(vaafi_headers = {})
     def all_documents
       get 'documents/getAllDocuments'
     end
+
+    def upload(file_name, file_body, claim_id, tracked_item_id)
+      post 'queuedDocumentUploadService/ajaxUploadFile', file_body do |req|
+        req.headers['Content-Type'] = 'application/octet-stream'
+        req.params['systemName'] = 'vets.gov'
```

Comment:
i added a constant to the base service class, let's use that here (`SYSTEM_NAME`)